h2o-py/h2o/frame.py

```diff
@@ -3938,25 +3941,25 @@ def convert_H2OFrame_2_DMatrix(self, predictors, yresp, h2oXGBoostModel, return_
         numerical and enum columns alone.  Note that H2O one-hot-encoding introduces a missing(NA)
         column. There can be NAs in any columns.
 
-        Follow the steps below to compare H2OXGBoost and native XGBoost:
+        Follow the steps below to compare H2OXGBoost and native XGBoost::
 
-        1. Train the H2OXGBoost model with H2OFrame trainFile and generate a prediction:
-        h2oModelD = H2OXGBoostEstimator(**h2oParamsD) # parameters specified as a dict()
-        h2oModelD.train(x=myX, y=y, training_frame=trainFile) # train with H2OFrame trainFile
-        h2oPredict = h2oPredictD = h2oModelD.predict(trainFile)
+            1. Train the H2OXGBoost model with H2OFrame trainFile and generate a prediction:
```

Comment:
if we use a single code block, the idea is to add a `#` in front of the numbered bullets: the benefit is that it makes the items look different from the code itself, and as mentioned earlier, this way the entire block can be copy/paster safely into a Python interpreter.